.github/actions/shellcheck/action.yml

```diff
@@ -0,0 +1,61 @@
+# This is a reusable action for running ShellCheck,
+# a linter for shell scripts (https://shellcheck.net).
+
+# For more context, see:
+# https://github.com/openedx/.github/blob/master/docs/decisions/0001-shellcheck.rst
+
+name: ShellCheck
+
+inputs:
+
+  root-dirs:
+    description: "Directories to search for .sh files, space-separated."
+    reqired: true
+
+  # We pin a reasonable default version here and will update it over time.
+  # However, repositories that are particuarly concerned about
+  # build stability should specify an override shellcheck-version
+  # and manage updates themselves.
+  shellcheck-version:
+    description: "ShellCheck version to install.Must be a tag or branch of https://github.com/koalaman/shellcheck."
+    required: false
+    default: "v0.9.0"
+
+  shellcheck-options:
+    description: "Command-line options to pass through to shellcheck."
+    required: false
+    default: ""
```

Comment:
It turns out that reusable workflow can't accept arguments. They can accept secrets, but these aren't secrets.

So, I turned this into an action instead.

Reply:
note to self: `install.Must` -> `install. Must`